enterprise/app/org/org_members.tsx

```diff
@@ -39,11 +39,16 @@ type State = {
   isRemoveLoading?: boolean;
 };
 
-const ROLE_LABELS: Record<grp.Group.Role, string> = {
-  [grp.Group.Role.UNKNOWN_ROLE]: "",
-  [grp.Group.Role.ADMIN_ROLE]: "Admin",
-  [grp.Group.Role.DEVELOPER_ROLE]: "Developer",
-};
+function getRoleLabel(role: grp.Group.Role): string {
```

Comment:
the typechecker didn't appreciate the missing entries for the new role values so I updated this to make the typechecker not break when we add new roles.